README.md

```diff
@@ -5,9 +5,19 @@ My Ansible Playbooks and Dependencies
 
 Currently just a work in progress. Soon will be able to spin up my default Ubuntu workstation from clean install to fully operational.
 
+Dependencies
+------------
+Ansible 1.7+ is required to run this playbook.
+
+In Ubuntu 14.04 this is currently a backport.  To determine the most recent version currently available issue the following command:
```

Comment:
Pip is generally the easiest way to get the latested Ansible versions. I say generally since most people probably do not have Pip installed already.

We could update this to mention pip install ansible, but not go into the details of installing Pip.